install/README.md

```diff
@@ -26,14 +30,38 @@ to see if your system is capable of running Kata Containers.
 > - When choosing a stable release, replace all `master` occurrences in the URLs
 > with a `stable-x.y` version available on the [download server](http://download.opensuse.org/repositories/home:/katacontainers:/releases:/).
 
-### Distros
+> **Notes on package source verification**:
+> - The Kata packages hosted on the download server are signed with GPG to ensure integrity and authenticity.
+>
+> - [This is the public key](....) used for signing the packages and [this is its fingerprint](...).
+>
+> - When prompted in the installation process, only trust these signing key and fingerprint.
+> Avoid disabling GPG checks when possible.
+
+### Distributions
+
+Setting up Kata Containers on your system is a three step procedure:
+1. Install the Kata Packages.
+2. Install a supported container runtime manager.
+3. Configure the container runtime to use `kata-runtime` as OCI runtime.
+
+The table below provides a list of the currently supported distributions.
+
+ - If your distribution is supported, you can easily install Kata packages [using kata-manager](...).
```

Comment:
Thanks for the review, I agree about this being hard to read. But I pushed it out early to gather feedbacks like yours :)

My first attempt was to have the different options available in a single document. Also, I felt it was worth documenting how to use `kata-manager` and not just mention that it can be used as an option.

Let's attempt to structure it with subsections.

Reply:
Hi @marcov - sure and raising PRs like this is **much** appreciated! Please don't misunderstand - I think this is moving in the right direction but delicate doc rework like this can take a few iterations. It's also just as difficult as code refactoring imho (but arguably more important! ;)

I think it makes a lot of sense having all the options in the top-level README - we just need to make some tweaks to make it clear to both "power users" and "newbies" which option is most appropriate for them :)

It would be good in fact to get more input on this from the rest of the team and @klynnrif and @intelkevinputnam.

Reply:
hehe @jodh-intel please don't misunderstand me too :laughing: I agree 100% with what you said. 
What I meant is that I started with what looked like obvious to change and then this got out of hand.


`install/README.md` is the front page of Kata for the new users and it is important to get it right.

Maybe we can open the discussion to the ML or the next architecture call?

Reply:
@jodh-intel what do you think about this structure? 

I am not sure if it's better to name the section with the distributions table *Supported Distribution* or *Manual Installation*

* [Prerequisites](#)
* [Installation on a Linux system](#)
    + [Automated Installation](#)
    + [Scripted Installation](#)
    + [Supported Distributions / Manual Installation](#)
* [Installation on a Cloud Service Platform](#)
* [Further information](#further-information)

Reply:
Nice - I like it.